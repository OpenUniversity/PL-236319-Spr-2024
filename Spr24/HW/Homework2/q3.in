print "===TEST START===\n";
val match1 = isMatch "*a*b*" "xaybz";  
val match2 = isMatch "ab*c" "abcd";    
val match3 = isMatch "a*b*c" "aabbc";  
val match4 = isMatch "abc*" "abcde";   
val match5 = isMatch "a*c" "abbbbbbc"; 
val match6 = isMatch "a*b*c*d*" "abcd";