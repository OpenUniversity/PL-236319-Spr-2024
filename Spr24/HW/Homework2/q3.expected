val it = () : unit
val match1 = true : bool
val match2 = false : bool
val match3 = true : bool
val match4 = true : bool
val match5 = true : bool
val match6 = true : bool
- 
